src/go/reporter/reporter.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2021 The Magma Authors.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package reporter
```

Comment:
nit: maybe we can name this package `crash` or something else; `reporter` is somewhat generic and means a lot of different things around magma

Reply:
yeah will rename